tests/README.md

```diff
@@ -0,0 +1,137 @@
+This readme is a manual on how to get started with unit testing on javascript and nodejs
+
+Original author: BrainDoctor (github), July 2017
+
+# Installation
+
+Tested on Linux Mint 18.2 Sara (Ubuntu/debian derivative)
+
+Make sure you are the user who is developer (permissions, except sudo ofc)
+
+```sh
+sudo apt-get install nodejs npm chromium-browser
+
+cd /path/to/your/netdata
+npm install
+cd node_modules
+git clone https://github.com/BrainDoctor/jasmine-node.git
```

Comment:
This dependency can also be added to package.json.

Check the documentation https://docs.npmjs.com/files/package.json#git-urls-as-dependencies